backends/graphs/README.md

```diff
@@ -31,13 +70,18 @@ test program:
 
 ![Flowlet switching ingress graph](resources/flowlet_switching-bmv2.ingress.png)
 
+Here is the graph generated for the egress control block of the
+[flowlet_switching-bmv2.p4](../../testdata/p4_16_samples/flowlet_switching-bmv2.p4)
+test program:
+
+![Flowlet switching egress graph](resources/flowlet_switching-bmv2.egress.png)
```

Comment:
some of the displayed nodes are very wide, is there a way to fix that?

Reply:
I can set limit for length of string to some reasonable number, like 25 chars?

Was unsure whether too lengthy nodes are a problem, as they are also created by old solution.